### PR TITLE
Remove Lottie

### PR DESCRIPTION
**Pull request by yurii-glia**
Replacing operator typing indicator with native CoreAnimation.
MOB-2267

### CODE REVIEW

File: GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift [hunk 1]
```diff
@@ -1,73 +1,159 @@
...
-        didSet {
-            isHidden ? animationView.stop() : animationView.play()
-        }
-    }
-    private let style: OperatorTypingIndicatorStyle
-    private let animationView = AnimationView()
-    private let kViewSize = CGSize(width: 28, height: 28)
-    private let kLeftInset: CGFloat = 10
-    private let bundleManaging: BundleManaging
+    lazy var dotLeadingView = DotView(
```
Comment (EgorovEI) [inline GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift:6]: Could we make dots `private`?

File: GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift [hunk 2]
```diff
@@ -1,73 +1,159 @@
...
     }
 
     private func updateAccessibility() {
         accessibilityLabel = style.accessibility.label.withOperatorName(accessibilityProperties.operatorName)
     }
 }
 
+extension OperatorTypingIndicatorView {
+    final class DotView: BaseView {
+        let color: UIColor
```
Comment (EgorovEI) [inline GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift:73]: the same question here

**Review by EgorovEI — APPROVED** [review]

File: GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift (hunk 1 above)
Comment (yurii-glia) [reply GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift]: fixed - a442c4ff7e7aeb045efb220807aee5e31d8fd6b9

File: GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift (hunk 2 above)
Comment (yurii-glia) [reply GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift]: fixed - a442c4ff7e7aeb045efb220807aee5e31d8fd6b9

**Review by igorkravchenko — APPROVED** [review]

**Comment by yurii-glia** [comment]
!squash

**Review by gersonnoboa — APPROVED** [review]
But there is a SwiftLint warning

**Comment by yurii-glia** [comment]
> But there is a SwiftLint warning

Swiftlint warnings have been fixed.

**Pull request merged**